.travis.yml

```diff
@@ -1,7 +1,34 @@
-sudo: false
-language: node_js
-node_js:
-  - 10
-  - 11
-script:
-  - npm run build && npm test
+language: rust
```

Comment:
We are in the process of migrating the crates.io repo to GitHub Actions instead of Travis. We probably want to use GitHub Actions here as well in the long run.

Reply:
Of course! Happy to do that in the long run.